src/Collections-Sequenceable/Interval.class.st

```diff
@@ -369,6 +369,13 @@ Interval >> size [
 				ifFalse: [^ stop - start // step + 1]]
 ]
 
+{ #category : #accessing }
+Interval >> sort: aBlock [
+	"What sorting an Interval means is not clear."
+
+	self error: 'you can not sort an interval'
```

Comment:
Except for the missing capital letter at the beginning of the error message and the dot at the end, this is fine for me. :-)

Reply:
I wrote it like that to be consistent with the Interval>>at:put: method:
```at: anInteger put: anObject 
	"Storing into an Interval is not allowed."

	self error: 'you can not store into an interval'```

Reply:
But sure, why not.